lib/auth/access.go

```diff
@@ -97,6 +98,12 @@ func (a *Server) UpsertRole(ctx context.Context, role types.Role) (types.Role, e
 	return upserted, nil
 }
 
+var (
+	errDeleteRoleUser       = trace.BadParameter("failed to delete a role that is still in use by a user, check the system server logs for more details")
```

Comment:
Looks better, last suggestion (up to you to accept or not) would be to use `errors.New` here.

We don't really need to embed a stack trace in these errors, since it won't point to anything interesting.